hackerank/js-practice/camelCase.js

```diff
@@ -0,0 +1,11 @@
+function camelcase(s) {
+   let count = 1;
+   for(let i in s){
+    if(s.charAt(i) === s.charAt(i).toUpperCase()){
+      count++
```

Comment:
I recommend using a code style plugging to be more consistent in your code styling.
- add `;` at the of line or not
- add space before `{` 
- add space after `(` 

These seem the most used:
- [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify)
- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode )